android-animation-arsenal/src/main/java/com/android_animation_arsenal/AnimationArsenal.java

```diff
@@ -40,15 +37,14 @@
     public static Transition getExplodeTransition(Transition.TransitionListener listener, int
             duration)
     {
-        if(Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP)
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
         {
-            return null;
+            Explode explode = new Explode();
+            explode.setDuration(duration);
+            explode.addListener(listener);
+            return explode;
         }
-
-        Explode explode = new Explode();
-        explode.setDuration(duration);
-        explode.addListener(listener);
-        return explode;
+        return null;
```

Comment:
I think you should still add `@TargetApi(Build.VERSION_CODES.LOLLIPOP)` to the top of this method :smile_cat:.